### PR TITLE
Switch to using the openshift downloader image

### PR DESCRIPTION
**Pull request by derekhiggins**
Its does the same thing as the openshift version, so we're gonna
drop the metal3 one.

### CODE REVIEW

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1219/

File: 04_setup_ironic.sh [hunk 1]
```diff
@@ -67,6 +69,8 @@ fi
 # Create pod
 sudo podman pod create -n ironic-pod 
 
+IRONIC_RHCOS_DOWNLOADER_IMAGE=$(oc adm release info --registry-config $REGISTRY_AUTH_FILE $OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE --image-for=ironic-rhcos-downloader)
+
```
Comment (hardys) [inline 04_setup_ironic.sh]: CI seems to be failing on this like:

```
error: info expects one argument, or a connection to an OpenShift 4.x server: invalid configuration: Missing or incomplete configuration info.  Please login or point to an existing, complete config file:
```

Comment (derekhiggins) [reply 04_setup_ironic.sh]: Y

> CI seems to be failing on this like:
> 
> ```
> error: info expects one argument, or a connection to an OpenShift 4.x server: invalid configuration: Missing or incomplete configuration info.  Please login or point to an existing, complete config file:
> ```

ya, I just noticed this this morning, I'm testing a new version at the moment, should be up soon.

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1238/

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1257/

**Comment by metal3ci** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/1258/

**Pull request merged**